cmd/api/src/daemons/datapipe/jobs.go

```diff
@@ -160,17 +176,25 @@ func (s *Daemon) preProcessIngestFile(path string, fileType model.FileType) ([]s
 	}
 }
 
-func (s *Daemon) processIngestFile(ctx context.Context, path string, fileType model.FileType) error {
+// processIngestFile reads the files at the path supplied, and returns the total number of files in the
+// archive, the number of files that failed to ingest as JSON, and an error
+func (s *Daemon) processIngestFile(ctx context.Context, path string, fileType model.FileType) (int, int, error) {
 	if paths, err := s.preProcessIngestFile(path, fileType); err != nil {
-		return err
+		return len(paths), 0, err
```

Comment:
Why return len(paths) here? If the zip file fails to process, do we even get anything back from this?